README.md

```diff
@@ -309,7 +308,7 @@ title: {
   - `continent`: Continent name that came from the GeoJSON map file.
 - `longitudeKey:` _(string, default: 'longitude')_ Column name of the longitude (from the dataset).
 - `latitudeKey:` _(string, default: 'latitude')_ Column name of the latitude (from the dataset).
-- `exportSvg:` _(string, default: null)_ URL endpoint to download the current visualization as SVG. Read more on the section export SVG. (more details on a the section "Export as SVG")
+- `exportSvg:` _(bool, default: false)_ Show button to download the current visualization as SVG. You'll need to import [FileSaver.js](https://github.com/eligrey/FileSaver.js) to make it work
```

Comment:
I didn't quite follow the "you'll need to import" instructions at first.  Maybe change to:

> - `exportSvg:` _(bool, default: false)_ Show button to download the current visualization as SVG. You'll need to  download [FileSaver.js](https://github.com/eligrey/FileSaver.js) and add a `<script src="filesaver.min.js">` to make it work.  You may also use the [CDN version](https://cdnjs.com/libraries/FileSaver.js) in your `<script>` source.